matrix-sdk-android/src/main/java/fr/gouv/tchap/android/sdk/api/session/accountdata/HideProfileContent.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.gouv.tchap.android.sdk.api.session.accountdata
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class HideProfileContent(
+        @Json(name = "hide_profile") val hideProfile: Boolean
+)
```

Comment:
This can be defined on the app, but not in the SDK

Reply:
After thinking, we prefer add this definition in the Tchap sdk level